src/Scripting/Core/ScriptBuilder.cs

```diff
@@ -115,20 +115,28 @@ private static void ThrowIfAnyCompilationErrors(DiagnosticBag diagnostics, Diagn
         /// </summary>
         private Func<object[], Task<T>> Build<T>(
             Compilation compilation,
-            DiagnosticBag diagnostics,
+            DiagnosticBag diagnostics, ScriptOptions scriptOptions,
```

Comment:
I'd just pass DebugInformationFormat to this method because that's the only thing needed here.

---
In reply to: [96129988](https://github.com/dotnet/roslyn/pull/16489#discussion_r96129988) [](ancestors = 96129988)